### DISCUSSION TITLE
Updating Config

### DISCUSSION THREAD

**Discussion by mhuryasar** [erossini/BlazorChartjs#34] [Q&A]
Sorry, I'm new to ChartJS. I'm trying to add/remove datasets dynamically, but the updated _config doesn't reflect in the chart. What do I need to do to reflect the new _config in the chart?

**Comment by erossini** [comment]
Have you check the demo website or the example [on my blog](https://www.puresourcecode.com/dotnet/blazor/blazor-component-for-chartjs/)? 

Every time you update the `_config` the chart is updated immediately. There is no `update` function, for now.
